app/helpers/search_helper.rb

```diff
@@ -183,12 +183,12 @@ def value_for_sign_attribute(number, attribute, main)
   end
 
   def classes_for_sign_attribute(attribute, main)
-    classes = 'image rounded'
-    if main
-      classes += 'main_image'
-    elsif attribute == :handshape
-      classes += 'transition'
-    end
+    # a space is required after the base class names below to ensure that they are
+    # properly separated from other classes assigned in the conditionals.
+    classes = 'image rounded '
+    classes += 'main_image' if main
```

Comment:
Surely a space after each class? Or push the classes onto an array and join with spaces at the end of the method?